### ISSUE TITLE
Errors while closing the player tab

### ISSUE THREAD

**Issue by kalwalt** [augmentmy-world/arStudio#12]
I receive in the console this error while closing the video with the stop button and switching from player tab to scene:
```
arcontroller.component.js?version=0.64:207 Uncaught TypeError: Cannot read property 'clientWidth' of undefined
```

and also:

```
timeline.js?version=0.64:376 Uncaught TypeError: Cannot read property 'duration' of null
    at Timeline.updateTimelineData (timeline.js?version=0.64:376)
    at Timeline.onMouseWheel (timeline.js?version=0.64:1625)
```

**Comment by kalwalt** [comment]
The second error message happens also in **webglstudio** not only in **arStudio**!

**Comment by kalwalt** [comment]
The first message erro should be fixed with Pr #19, to be confirmed for the second one.

**Comment by kalwalt** [comment]
I think that we can close also this issue. it was solved with PR #28
